forch/forchestrator.py

```diff
@@ -628,6 +629,12 @@ def _validate_config(self, config):
                 warnings.append(('faucet_dp_mac', 'faucet_dp_mac for DPs are not identical'))
         return warnings
 
+    def _update_config_warning_varz(self):
+        self._metrics.update_var(
+            'faucet_config_warning_count', len(self._config_summary.warnings))
+        for warning_key, warning_msg in self._config_summary.warnings.items():
+            self._metrics.update_var('faucet_config_warning', 0, [warning_key, warning_msg])
```

Comment:
should be 1 not 0, yes, there's no case where it's anything, but a 0 value implies "this is not true" which is not the case.

Reply:
Makes sense. Changed to 1